### PR TITLE
Fix links to old wiki

### PR DESCRIPTION
**Pull request by dankcushions**

### CODE REVIEW

File: docs/Add-a-New-System-in-EmulationStation.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Say that you now want to add the SNES rom hack [Super Metroid: Redesign](http://
 
 Add .smc and .SMC (or any relevant extensions) to the `<extension>` tag in your custom `es_systems.cfg file `and restart EmulationStation.
 
-To launch Super Metroid: Redesign from EmulationStation, you will need to bring up the [runcommand menu](https://github.com/retropie/retropie-setup/wiki/runcommand) by pressing any key on a keyboard or button 0 on your controller before the game begins. In the menu, choose the lr-snes9x2010 as the emulator for your rom and then launch.
+To launch Super Metroid: Redesign from EmulationStation, you will need to bring up the [runcommand menu](runcommand) by pressing any key on a keyboard or button 0 on your controller before the game begins. In the menu, choose the lr-snes9x2010 as the emulator for your rom and then launch.
```
Comment (cmitu) [inline docs/Add-a-New-System-in-EmulationStation.md:124]: `(runcommand)` should be `(Runcommand)`.

File: docs/Bluetooth-Controller.md [hunk 2]
```diff
@@ -43,8 +43,7 @@ You will get a window popping up telling you if the connection was successful or
 4. Buy a new BT adapter and put it to your Raspberry. Add `dtoverlay=pi3-disable-bt` in `/boot/config.txt` to disable internal BT module (ultima ratio, speeds up communication, extents range and speed depending on your BT adapter model)
 
 ## Manual Bluetooth Configuration
-## !!Please note this is for pre v4.0 RetroPie - New 8bitdo details here!!:
-https://github.com/RetroPie/RetroPie-Setup/wiki/8bitdo-Controller
+## !!Please note this is for pre v4.0 RetroPie - New 8bitdo details [here](8bitdo-Controller)!!
```
Comment (cmitu) [inline docs/Bluetooth-Controller.md:46]: `(8bitdo-Controller)` should be `(8Bitdo-Controller)`

File: docs/Doom.md [hunk 3]
```diff
@@ -402,7 +402,7 @@ Having completed this step, you will now want to restart Emulation Station, navi
 
 lr-prboom utilises normal Retroarch control configurations found in the RetroArch menu by pressing 'Hotkey Enable+X' on your controller after the game software has launched. From the RetroArch Quick Menu, navigate to 'Controls'.
 
-For more information on custom RetroArch controls see: [RetroArch Configuration](https://github.com/retropie/retropie-setup/wiki/retroarch-configuration)
+For more information on custom RetroArch controls see: [RetroArch Configuration](retroarch-configuration)
```
Comment (cmitu) [inline docs/Doom.md:405]: `[RetroArch Configuration](retroarch-configuration)` should be `[RetroArch Configuration](RetroArch-Configuration)`.

File: docs/KODI.md [hunk 4]
```diff
@@ -74,7 +74,7 @@ If you are experiencing problems with your Xbox controls, see the following exam
 
 ## Kodi as its own system instead of in ports
 
-The first method here is a cleaner method that won't mess with the RetroPie setup script updates. (Note: in the future, you will have to manually update `/home/pi/.emulationstation/es_systems.cfg`, as it will not be overwritten by RetroPie updates.) Once you've installed Kodi from the optional menu of the setup script, drop into a terminal with F4 or access the pi over [SSH](https://github.com/retropie/retropie-setup/wiki/ssh).
+The first method here is a cleaner method that won't mess with the RetroPie setup script updates. (Note: in the future, you will have to manually update `/home/pi/.emulationstation/es_systems.cfg`, as it will not be overwritten by RetroPie updates.) Once you've installed Kodi from the optional menu of the setup script, drop into a terminal with F4 or access the pi over [SSH](ssh).
```
Comment (cmitu) [inline docs/KODI.md:77]: `[SSH](ssh)` should be `[SSH](SSH)`.

File: docs/Mapping-a-Controller-for-Intellivision.md [hunk 5]
```diff
@@ -242,7 +242,7 @@ For games that require Intellivoice support, add an additional "jzintv-voice" li
 <pre>jzintv-voice = "/opt/retropie/emulators/jzintv/bin/jzintv -p /home/pi/RetroPie/BIOS -q --kbdhackfile=/home/pi/RetroPie/roms/intellivision/hackfile.cfg %ROM%" -v1</pre>
 For games that require Intellivision ECS support, add an additional "jzintv-ecs" line that includes the '-s1' parameter to `/opt/retropie/configs/intellivision/emulators.cfg` per below:
 <pre>jzintv-ecs = "/opt/retropie/emulators/jzintv/bin/jzintv -p /home/pi/RetroPie/BIOS -q --kbdhackfile=/home/pi/RetroPie/roms/intellivision/hackfile.cfg %ROM%" -s1</pre>
-Note that games requiring Intellivoice and/or ECS support must be assigned to the lines defined above using the [Runcommand Launch Menu](https://github.com/RetroPie/RetroPie-Setup/wiki/runcommand).
+Note that games requiring Intellivoice and/or ECS support must be assigned to the lines defined above using the [Runcommand Launch Menu](runcommand).
```
Comment (cmitu) [inline docs/Mapping-a-Controller-for-Intellivision.md:245]: `[Runcommand Launch Menu](runcommand)` should be `[Runcommand Launch Menu](Runcommand#runcommand-launch-menu)`.

File: docs/Netplay.md [hunk 6]
```diff
@@ -25,7 +25,7 @@ If you will be acting as the host...
 - Either go to your router settings and open the port 55435 for both TCP and UDP **or** change the TCP/UDP Port setting in RetroArch to one that's already open
 - Select a number for Delay Frames. If you are experiencing a very low fps, try increasing this number.
 - Pick a Nickname
-- When launching a ROM to use netplay you'll open up the [runcommand menu](https://github.com/retropie/retropie-setup/wiki/runcommand) by pressing js0 (or any key on your keyboard) as your ROM loads and then you'll select launch netplay
+- When launching a ROM to use netplay you'll open up the [runcommand menu](runcommand) by pressing js0 (or any key on your keyboard) as your ROM loads and then you'll select launch netplay
```
Comment (cmitu) [inline docs/Netplay.md:28]: `[runcommand menu](runcommand)` should be `[Runcommand launch menu](Runcommand##runcommand-launch-menu)`.

File: docs/Netplay.md [hunk 7]
```diff
@@ -45,7 +45,7 @@ If you will be joining a game...
 - Change TCP/UDP Port to the number your friend gave you
 - Change Delay Frames to the number your friend gave you
 - Pick a Nickname
-- When launching a ROM to use netplay you'll open up the [runcommand menu](https://github.com/retropie/retropie-setup/wiki/runcommand) by pressing js0 (or any key on your keyboard) as your ROM loads and then you'll select launch netplay
+- When launching a ROM to use netplay you'll open up the [runcommand menu](runcommand) by pressing js0 (or any key on your keyboard) as your ROM loads and then you'll select launch netplay
```
Comment (cmitu) [inline docs/Netplay.md:48]: Same here - `[runcommand menu](runcommand)` -> `[Runcommand launch menu](Runcommand##runcommand-launch-menu)`.

File: docs/Optimization-for-Nintendo-64.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ then cancel, exit and reboot.
 
 ## Selecting the Correct Emulator and Graphics Plugin
 
-Just as important as overclocking, selecting the right emulator/graphics plugin from the [runcommand](https://github.com/RetroPie/RetroPie-Setup/wiki/runcommand) menu on a per game basis will also increase performance. Selecting the right plugin can make all the difference in making a game playable. 
+Just as important as overclocking, selecting the right emulator/graphics plugin from the [runcommand](runcommand) menu on a per game basis will also increase performance. Selecting the right plugin can make all the difference in making a game playable.
```
Comment (cmitu) [inline docs/Optimization-for-Nintendo-64.md:34]: `[runcommand](runcommand) menu` should be `[Runcommand launch menu](Runcommand##runcommand-launch-menu)`.

File: docs/Take-and-Scrape-Your-Own-Screenshots.md [hunk 9]
```diff
@@ -263,7 +263,7 @@ https://retropie.org.uk/forum/topic/3353/take-and-scrape-your-own-screenshots/
 
 https://github.com/RetroPie/EmulationStation/blob/master/GAMELISTS.md
 
-https://github.com/retropie/retropie-setup/wiki/scraper
+[Scraper](scraper)
```
Comment (cmitu) [inline docs/Take-and-Scrape-Your-Own-Screenshots.md:266]: `[Scraper](scraper)` should be `[Scraper](Scraper)`.

File: docs/lr-mame2003.md [hunk 10]
```diff
@@ -99,7 +99,7 @@ For users who have a rotatable display, this mode shows vertical games (e.g. Pac
 In addition to this, you will want to stop rotating the games and instead show them in along the length of the screen, `video_allow_rotate` in mame-libretro/arcade to false via the [Configuration Editor](Configuration-Editor.md).
 
 ## Save states
-Beyond [High scores](#high-scores), lr-mame2003 supports save states through the [usual Retroarch hotkeys](https://github.com/retropie/retropie-setup/wiki/retroarch-configuration#default-joypad-hotkeys). However, save state support has to be implemented per-driver, so they won't work with all games.
+Beyond [High scores](#high-scores), lr-mame2003 supports save states through the [usual Retroarch hotkeys](retroarch-configuration#default-joypad-hotkeys). However, save state support has to be implemented per-driver, so they won't work with all games.
```
Comment (cmitu) [inline docs/lr-mame2003.md:102]: `[usual Retroarch hotkeys](retroarch-configuration#default-joypad-hotkeys)` should be `[usual Retroarch hotkeys](RetroArch-Configuration#default-joypad-hotkeys)`.

**Comment by dankcushions** [comment]
Should all be fixed now :)

**Comment by joolswills** [comment]
Thanks.

**Comment by joolswills** [comment]
I'll regenerate the docs now. Might be a good idea to run a links checker on the docs. Pick up any broken external links too.

**Pull request merged**